### PR TITLE
Capture stderr when running commands

### PR DESCRIPTION
**Pull request by justinsb**
This is so we can start to diagnose some of the failures where kops times out while coming up e.g. https://github.com/kubernetes/kubernetes/issues/46370#issuecomment-305398342

cc @dchen1107

### CODE REVIEW

**Comment by justinsb** [comment]
I think this was supposed to be captured by WrapError, but 0c74e7b4dfb0b6263b67e9f057acad3277f55c1a wrapped the error in another error, and golang.

**Comment by krzyzacy** [comment]
/assign @fejta 

The idea is LG, not super familiar if this is the best way of dealing with stderr in go, add @fejta in case he has other thoughts.

File: kubetest/util.go [hunk 1]
```diff
@@ -217,8 +219,16 @@ func output(cmd *exec.Cmd) ([]byte, error) {
 		if !started {
 			return
 		}
-		b, err := cmd.Output()
-		finished <- result{b, err}
+
+		var stderr bytes.Buffer
+		if cmd.Stderr == nil {
```
Comment (fejta) [inline kubetest/util.go]: Seems like we can combine this with the `if verbose` statement?

**Comment by fejta** [comment]
Do we still want/need this? Seems like a good idea! If so please rebase and reopen the PR, thanks!
/close

**Pull request closed without merging**